### ISSUE TITLE
[chr11:130986347-130986524][DEL][SVLEN=177bp]

### ISSUE THREAD

**Issue by jzook** [jzook/testHG008curation#234]
`chr11	130986347	Minda_22	N	<DEL>	.	PASS	SVLEN=177;SVTYPE=DEL;SUPP_VEC=PB_ID_48883_2,PB_d_64,ONT_ID_38719_2,ONT_d_29,ILL_MantaDEL:187930:0:0:0:0:0,PB_severus_DEL1291,ONT_severus_DEL1402,ILL_2186766853:2,PB_Sniffles2.DEL.3A6MA,ONT_Sniffles2.DEL.3ABMA
`

https://v2.genomeribbon.com/?session=https://42basepairs.com/download/s3/giab-data/ribbon-json/ribbon-hg008-cov-bedpe.json&locus=chr11:130986347#ribbon

**Comment by Faizal-Eeman** [comment]
<img width="912" alt="Screenshot 2024-10-11 at 3 44 59 PM" src="https://github.com/user-attachments/assets/7f9ae3b5-51f0-40b3-93bf-4fae05b07dc9">
<img width="1199" alt="Screenshot 2024-10-11 at 5 17 39 PM" src="https://github.com/user-attachments/assets/76d6f169-36a7-4038-82d5-a3e504a28ab0">

- No other SVs/CNAs are seen in or around the region

**Comment by Faizal-Eeman** [comment]
<img width="1079" alt="Screenshot 2024-10-11 at 4 46 14 PM" src="https://github.com/user-attachments/assets/17734d5b-05f6-478b-a731-0e65c22b35e5">

- Het deletion seen in PacBio Tumor BAMs

**Comment by Faizal-Eeman** [comment]
<img width="1075" alt="Screenshot 2024-10-11 at 5 03 32 PM" src="https://github.com/user-attachments/assets/04feccd5-991c-4956-9a51-4893dfdce514">

- Start and End positions of this deletion appear to be correct
- Although this deletion appears to be correct, it lies in a repeat rich region

**Comment by Faizal-Eeman** [comment]
This variant however lies in a RefSeq Unannotated region. Therefore, flagging this variant for review.

**Comment by Faizal-Eeman** [comment]
<img width="710" alt="Screenshot 2024-10-13 at 10 56 11 PM" src="https://github.com/user-attachments/assets/e208b1a5-e469-46f0-81b2-5f00ef3d0d3f">

- Looks like the chr6-chr11 BND from #197 is ~6Mbp upstream of this small deletion
- However, the BND and deletion appear to be independent and do not overlap
- The average coverage seems to be 120X for PacBio tumor BAMs. The coverage for reads with deletion is ~50X and wild type reads also have ~50X coverage. This suggests the deletion event to be simple and not complex.

**Comment by jzook** [comment]
I agree that this seems to be unrelated to the translocation, and seems to be a clonal DEL in a normal diploid region of chr11
